services/deploymentService.js

```
@@ -0,0 +1,728 @@
+
+      // Install Heroku CLI if not present
+      await this.ensureHerokuCLI();
+
+      // Login to Heroku
+      await execAsync(`echo "${herokuToken}" | heroku auth:token`);
+
+      // Create Heroku app if not exists
+      if (!appName) {
+        const { stdout } = await execAsync('heroku create');
+        const appMatch = stdout.match(/https:\/\/[^-\s]+/);
+        appName = appMatch ? appMatch[0].replace('https://', '').replace('.herokuapp.com', '') : null;
```

Biome (2.1.2)

[error] Can't assign appName because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)